### PR TITLE
pointed bug report link to an info page instead of bugzilla

### LINKED ISSUE #61: Reporting a bug doesn't guide the user
In http://beta-ovirtweb.rhcloud.com/documentation/ Reporting a BUG link just point to the oVirt classification bugzilla without saying what to add to the report (like log-collector report, sos report, single log file). It would be better to add a page explaining how to create a useful bug report, with instructions about ovirt teams, projects, components, attachments.

### PR DESCRIPTION
**Pull request by garrett**
fixes #61

### CODE REVIEW

File: source/community/activities/outreachy.html.md [hunk 1]
```diff
@@ -71,7 +71,7 @@ Some ideas:
...
-2.  *Eliminate Vdsm network bugs*: We have many [link bugs](https://bugzilla.redhat.com/buglist.cgi?bug_status=NEW&bug_status=ASSIGNED&columnlist=short_desc%2Ccomponent%2Cbug_status%2Cflagtypes.name%2Cassigned_to&component=vdsm&f0=OP&f1=OP&f2=status_whiteboard&f3=CP&f4=CP&f5=component&j1=OR&known_name=net&list_id=2893489&o2=substring&o5=notsubstring&product=oVirt&product=Red%20Hat%20Enterprise%20Linux%206&product=Red%20Hat%20Enterprise%20Virtualization%20Manager&query_based_on=net&query_format=advanced&v2=network&v5=Guide). I'd love to sqush them all, big and small, simple and complex.
+2.  *Eliminate Vdsm network bugs*: We have many [link bugs](https://bugzilla.redhat.com/buglist.cgi?bug_status=NEW&bug_status=ASSIGNED&columnlist=short_desc%2Ccomponent%2Cbug_status%2Cflagtypes.name%2Cassigned_to&product=vdsm&f0=OP&f1=OP&f2=status_whiteboard&f3=CP&f4=CP&f5=component&j1=OR&known_name=net&list_id=2893489&o2=substring&o5=notsubstring&classificiation=oVirt&product=Red%20Hat%20Enterprise%20Linux%206&product=Red%20Hat%20Enterprise%20Virtualization%20Manager&query_based_on=net&query_format=advanced&v2=network&v5=Guide). I'd love to sqush them all, big and small, simple and complex.
```
Comment (garrett) [inline source/community/activities/outreachy.html.md]: I'm not sure exactly what the new query should be here. (I do know the old one is broken.)

File: source/develop/developer-guide/vdsm/todo.html.md [hunk 2]
```diff
@@ -121,6 +121,6 @@ We intend to move this TODO into this [Trello board](https://trello.com/b/U3lsbV
...
+*   pick one of the [<https://bugzilla.redhat.com/buglist.cgi?action=wrap&bug_file_loc>=&bug_file_loc_type=allwordssubstr&bug_id=&bug_id_type=anyexact&chfieldfrom=&chfieldto=Now&chfieldvalue=&product=vdsm&deadlinefrom=&deadlineto=&email1=&email2=&emailtype1=substring&emailtype2=substring&field0-0-0=flagtypes.name&keywords=&keywords_type=allwords&longdesc=&longdesc_type=allwordssubstr&short_desc=&short_desc_type=allwordssubstr&status_whiteboard=&status_whiteboard_type=allwordssubstr&type0-0-0=notsubstring&value0-0-0=rhel-6.2.0&votes=&=&bug_status=NEW NEW bugs], post a patch to [gerrit](http://gerrit.ovirt.org), and make the bug yours.
```
Comment (garrett) [inline source/develop/developer-guide/vdsm/todo.html.md]: I'm not sure what the proper link to the query should be here. Both old and this new one are wrong too.

File: source/develop/release-management/features/infra/python-sdk.html.md [hunk 3]
```diff
@@ -284,7 +284,7 @@ codegen
 
 ### Bugs/RFEs
 
-[Bugzila](https://bugzilla.redhat.com/buglist.cgi?list_id=363325&classification=Community&query_based_on=ovirt_sdk_bugs&query_format=advanced&token=1343748821-516898cbb4251bbe2a7856f547f55f74&bug_status=NEW&bug_status=ASSIGNED&component=ovirt-engine-sdk&product=oVirt&known_name=ovirt_sdk_bugs)
+[Bugzila](https://bugzilla.redhat.com/buglist.cgi?list_id=363325&classification=Community&query_based_on=ovirt_sdk_bugs&query_format=advanced&token=1343748821-516898cbb4251bbe2a7856f547f55f74&bug_status=NEW&bug_status=ASSIGNED&component=ovirt-engine-sdk&classificiation=oVirt&known_name=ovirt_sdk_bugs)
```
Comment (garrett) [inline source/develop/release-management/features/infra/python-sdk.html.md]: Another improper bugzilla link.

File: source/community/about/community-guidelines.html.md [hunk 4]
```diff
@@ -73,7 +73,7 @@ In general, we try to follow the [Wikipedia Style Guide](http://en.wikipedia.org
 
 ## Bug reporting guidelines
 
-*   Before reporting your bug, please [search the bug tracker](https://bugzilla.redhat.com/query.cgi?product=oVirt) to see if it has been reported already. A comment added to an existing bug report is also very useful.
+*   Before reporting your bug, please [search the bug tracker](https://bugzilla.redhat.com/query.cgi?classificiation=oVirt) to see if it has been reported already. A comment added to an existing bug report is also very useful.
```
Comment (garrett) [inline source/community/about/community-guidelines.html.md]: Close, but should probably be:
https://bugzilla.redhat.com/buglist.cgi?bug_status=__open__&classification=oVirt

**Comment by sandrobonazzola** [comment]
:+1:

**Comment by thatdocslady** [comment]
@sandrobonazzola @garrett if nothing else is required here, please implement and merge

**Pull request merged**